api/src/main/java/org/apache/cloudstack/api/command/admin/network/CreateNetworkOfferingCmd.java

```diff
@@ -94,7 +94,6 @@
 
     @Parameter(name = ApiConstants.SUPPORTED_SERVICES,
             type = CommandType.LIST,
-            required = true,
```

Comment:
Is there any update on the API command [createNetworkOffering](https://cloudstack.apache.org/api/apidocs-4.15/apis/createNetworkOffering.html) that I am not aware of?
As far as I know, `supportedservices` is required.

Reply:
For L2 network offerings, there are cases when there are no services, and an empty list is sent, so marking it as not required

Reply:
Aha, looks like a legacy restriction that no longer is necessary.
@davidjumani thanks for the clarification  :+1: